internal/pkg/manifest/env.go

```diff
@@ -102,9 +102,56 @@ type environmentNetworkConfig struct {
 }
 
 type environmentVPCConfig struct {
-	ID      *string              `yaml:"id,omitempty"`
-	CIDR    *IPNet               `yaml:"cidr,omitempty"`
-	Subnets subnetsConfiguration `yaml:"subnets,omitempty"`
+	ID                  *string              `yaml:"id,omitempty"`
+	CIDR                *IPNet               `yaml:"cidr,omitempty"`
+	Subnets             subnetsConfiguration `yaml:"subnets,omitempty"`
+	SecurityGroupConfig securityGroupConfig  `yaml:"security_group,omitempty"`
```

Comment:
Do we consider environment security group a part of VPC config? From my understanding, it should be on its own.

Reply:
I have took this field out of the VPC as per discussion and setting it on Network level instead under the VPC.